src/Uno.UI/UI/Xaml/Controls/CommandBar/CommandBar.Partial.cs

```diff
@@ -71,6 +71,9 @@ private enum OverflowInitialFocusItem
 		private readonly SerialDisposable m_overflowPopupClosedEventHandler = new SerialDisposable();
 		private readonly SerialDisposable m_overflowPresenterItemsPresenterKeyDownEventHandler = new SerialDisposable();
 
+		// Uno specific
+		private readonly SerialDisposable m_contentRootRenderTransformChangedEventHandler = new SerialDisposable();
```

Comment:
For now, these changes that are related to invalidating popup panel arrange can go into a separate PR. Also, I'd prefer us to open an issue to track removing this Uno-specific code